build/validate.sh

```diff
@@ -102,9 +97,5 @@ if [ "$BUILDMSG" == '' ]; then
 	BUILDSTATUS=success
 fi
 
-if [ "$TRAVIS" != '' ]; then
```

Comment:
What's the thinking behind removing the status reporting ? Have we replaced it elsewhere ?

Reply:
The Travis integration should already do that for us: https://blog.travis-ci.com/2018-05-07-announcing-support-for-github-checks-api-on-travis-ci-com

IIRC, setting the status with an API call was required before Travis moved to Github apps.